srcpkgs/nodejs-lts-10/template

```diff
@@ -31,38 +31,42 @@ replaces="iojs>=0"
 conflicts="nodejs nodejs-lts"
 provides="nodejs-runtime-0_1"
 
-if [ "$XBPS_WORDSIZE" -ne "$XBPS_TARGET_WORDSIZE" ]; then
-	nocross="host and target must have the same pointer size"
+if [ ! "$XBPS_WORDSIZE" == "$XBPS_TARGET_WORDSIZE" ]; then
```

Comment:
please replace this with a simple `"$XBPS_WORDSIZE" != "$XBPS_TARGET_WORDSIZE"``

Reply:
why replace it at all? the previous one is fine

Reply:
yeah, indeed. Didn't see it